src/components/TrailBlazerModal.tsx

```diff
@@ -8,6 +8,9 @@ import { VerticalTimeline, VerticalTimelineElement } from "react-vertical-timeli
 import { Loader } from "components";
 import { IFrameModal } from "./iFrameModal";
 import { TwModal } from "./Modal/TwModal";
+import ReactModal from "react-modal";
+
+const ModelR18 = Modal as ComponentType<ReactModal['props']>;
```

Comment:
I will have to come back with the implications of this change once I can thoroughly test

Reply:
This is not required. I deleted this.